src/Stepper/Stepper.jsx

```diff
@@ -0,0 +1,306 @@
+import React, {PropTypes} from 'react';
+
+import {getMuiTheme} from '../styles';
+import Paper from '../paper';
+
+const Stepper = React.createClass({
+  propTypes: {
+
+    /**
+     * The current active step index which passed by parent component.
+     */
+    activeStepIndex: PropTypes.number,
+
+    /**
+     * Children should be Step type.
+     */
+    children: PropTypes.node,
+
+    /*
+     * Override inline-style of the content container.
+     */
+    containerStyle: PropTypes.object,
+
+    /**
+     * Create suitable icon for step base on state of the step.
+     */
+    createIcon: PropTypes.func,
```

Comment:
Not sure about the name of this prop. `stepIcon`? `requestStepIcon`?

Could you update the description to the standard form: `Callback function that is fired when...`

And also add the callback signature tag mentioned previously:
https://github.com/callemall/material-ui/wiki/Documentation#callback-signature
 
Same for any other function props that are part of the public API.

Also, there's a new @returns tag that isn't documented in the wiki, but is relevant here (and possibly other function props. https://github.com/callemall/material-ui/pull/3542

Thanks!